src/libs/ReportActionsUtils.ts

```diff
@@ -586,18 +587,11 @@ function isConsecutiveActionMadeByPreviousActor(reportActions: ReportAction[] |
 /**
  * Checks if a reportAction is deprecated.
  */
-function isReportActionDeprecated(reportAction: OnyxEntry<ReportAction>, key: string | number): boolean {
+function isReportActionDeprecated(reportAction: OnyxEntry<ReportAction>): boolean {
     if (!reportAction) {
         return true;
     }
 
-    // HACK ALERT: We're temporarily filtering out any reportActions keyed by sequenceNumber
-    // to prevent bugs during the migration from sequenceNumber -> reportActionID
-    if (String(reportAction.sequenceNumber) === key) {
```

Comment:
We can't safely remove this yet. I checked logs and it's still being hit. Please add it back and suppress the deprecation warning